.gitlab-ci.yml

```diff
@@ -329,6 +367,42 @@ build/cuda101/intel/cuda/debug/static:
     BUILD_OMP: "ON"
     BUILD_CUDA: "ON"
     BUILD_TYPE: "Debug"
+    BUILD_SHARED_LIBS: "OFF"
+    CUDA_ARCH: 35
+  tags:
+    - private_ci
+    - cuda
+    - gpu
+
+# clang-cuda with cuda 10.1 and friends
+build/clang-cuda101/gcc/all/release/shared:
```

Comment:
No debug /static builds ?

Reply:
I had some initially, but removed them because clang-cuda really takes the "no optimizations" thing to the limit. But now that we only have two jobs, I guess it couldn't hurt.

Reply:
I just realized that one of the jobs was mis-named anyways, it was a static build after all

Reply:
Yeah, I think it should be fine. Unless it does really take the same time as 5 other equivalent jobs.

Reply:
It's really not that bad, the test only takes 3x the usual time.